arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,7 +410,7 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
-        "provisioningState": {
+        "state": {
```

Comment:
This is a standard property as per ARM convention. Did you get an approval by the ARM Review board? This looks incorrect to me.
/cc @ravbhatnagar - Can you please take a look at this issue?